looker/definitions/ads.toml

```diff
@@ -67,3 +73,11 @@ description = """
 columns_as_dimensions = true
 submission_date_column = "submission_date"
 client_id_column = "NULL"
+
+[data_sources.ppa_measurements]
+from_expression = "mozdata.ads.ppa_measurements"
+client_id_column = "NULL"
+friendly_name = "PPA Measurements"
+description = "Aggregated conversion data from Privacy-Preserving Attribution (PPA)"
+columns_as_dimensions = true
+submission_date_column = "DATE(collection_time)"
```

Comment:
as it is right now, this needs to be a column name. You could edit the `from_expression` and do the date conversion there.

Reply:
Didn't realize, thanks!